server-phoenix/lib/helios/application.ex

```diff
@@ -19,6 +19,7 @@ defmodule Helios.Application do
       # Start the Ecto repository
       Helios.Repo,
       # Start the Telemetry supervisor
+      {Task.Supervisor, name: SlackImageDownloader},
```

Comment:
nit: commit message is calling this a `rename` but it seems like its a new thing. Also not used yet?